### PR TITLE
Pipeline fixes

### PR DESCRIPTION
**Pull request by bettinaheim**
Fixes https://github.com/NVIDIA/cuda-quantum/issues/2409

### CODE REVIEW

**Review by 1tnguyen — APPROVED** [review]
LGTM 👍

File: runtime/nvqir/cutensornet/CMakeLists.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@
 enable_language(CUDA)
 find_package(CUDAToolkit REQUIRED)
 
-find_library(CUTENSOR_LIB
+find_library(CUTENSOR_LIB libcutensor.so.2
     NAMES   cutensor
```
Comment (boschmitt) [inline runtime/nvqir/cutensornet/CMakeLists.txt:14]:
```suggestion
find_library(CUTENSOR_LIB
    NAMES   cutensor libcutensor.so.2
```

**Pull request merged**